docs/sources/flow/reference/cli/run.md

```diff
@@ -50,6 +50,7 @@ The following flags are supported:
 * `--cluster.discover-peers`: List of key-value tuples for discovering peers (default `""`). Mutually exclusive with `--cluster.join-addresses`.
 * `--cluster.rejoin-interval`: How often to rejoin the list of peers (default `"60s"`).
 * `--cluster.advertise-address`: Address to advertise to other cluster nodes (default `""`).
+* `--cluster.advertise-interfaces`: List of interfaces used to infer an address to advertise. The first one available in the list will be selected (default `["eth0", "en0"]`).
```

Comment:
IMO the default shown here should be specified as the user would specify it (either comma-delimited or with multiple instances of the flag):

```suggestion
* `--cluster.advertise-interfaces`: List of interfaces used to infer an address to advertise. The first one available in the list will be selected (default `"eth0,en0"`).
```